rs/ic_os/network/src/mac_address.rs

```diff
@@ -124,6 +124,10 @@ pub fn generate_mac_address(
 /// Retrieves the MAC address from the IPMI LAN interface
 pub fn get_ipmi_mac() -> Result<FormattedMacAddress> {
     let output = Command::new("ipmitool").arg("lan").arg("print").output()?;
+
+    // A bug in our version of ipmitool causes it to exit with an error
+    // status, but we have enough output to work with anyway.
+    // https://github.com/ipmitool/ipmitool/issues/388
```

Comment:
I had erroneously removed this comment in this PR: https://github.com/dfinity/ic/commit/41a9d9db7c4996303a936f4e34f8ac57c4b0d158#diff-08f09c038240d9c12ff41bd5917e48dde5f8e5a1ffbd55b2d17732c580c5cc82L136